asterius/src/Asterius/Main.hs

```diff
@@ -264,6 +265,7 @@ ahcLink task = do
          ]
       <> ["-optl--no-gc-sections" | not (gcSections task)]
       <> ["-optl--verbose-err" | verboseErr task]
+      <> ["-optl--pic" | pic task]
```

Comment:
The `pic` flag is unused by `ahc-ld`, and not present in `LoadTask` right? So I don't think we need to pass `--pic` to `ahc-ld` here.

Reply:
Ah, right, that should have been part of fd0b4d545973014eb58867e870e2b186c9dcb0e8 but I missed it, nice catch!

Reply:
Should be fixed now.